include/rcpputils/filesystem_helper.hpp

```diff
@@ -259,35 +260,6 @@ RCPPUTILS_PUBLIC uint64_t file_size(const path & p);
  */
 RCPPUTILS_PUBLIC bool exists(const path & path_to_check);
 
-
-/**
- * \brief Get a path to a location in the temporary directory, if it's available.
- *
- * This does not create any directories.
- * On Windows, this uses "GetTempPathA"
- * On non-Windows, this prefers the environment variable TMPDIR, falling back to /tmp
- *
- * \return A path to a directory for storing temporary files and directories.
- */
-RCPPUTILS_PUBLIC path temp_directory_path();
-
-/**
- * \brief Construct a uniquely named temporary directory, in "parent", with format base_nameXXXXXX
- *
- * The output, if successful, is guaranteed to be a newly-created directory.
- * The underlying implementation keeps generating paths until one that does not exist is found.
- * This guarantees that there will be no existing files in the returned directory.
- *
- * \param[in] base_name User-specified portion of the created directory
- * \param[in] parent_path The parent path of the directory that will be created
- * \return A path to a newly-created directory with base_name and a 6-character unique suffix
- *
- * \throws std::system_error If any OS APIs do not succeed.
- */
-RCPPUTILS_PUBLIC path create_temp_directory(
-  const std::string & base_name,
-  const path & parent_path = temp_directory_path());
-
 /**
```

Comment:
So we still need to keep `create_temp_directory`, since there are downstream users.  We want to change that to use `std::filesystem::path`.

But we don't need `temp_directory_path`, since that is provided by `std::filesystem` already.

Reply:
yeah, sorry I was to excited about removing code. Restored